packages/flutter/test/painting/image_cache_test.dart

```diff
@@ -128,5 +128,14 @@ void main() {
       expect(imageCache.currentSizeBytes, 256);
       expect(imageCache.maximumSizeBytes, 256 + 1000);
     });
+
+    test('decodedCacheRatio', () async {
+      // Has default value.
+      expect(imageCache.decodedCacheRatioCap, isNot(null));
+
+      // Can be set.
+      imageCache.decodedCacheRatioCap = 1.0;
+      expect(imageCache.decodedCacheRatioCap, 1.0);
+    });
   });
 }
```

Comment:
you could add a test that the various image providers end up calling into the binding (rather than calling dart:ui directly) by creating your own subclass of the binding in the test. there should be some existing tests you can crib from that do similar things.

Reply:
(or if you put the method on the image cache, you just need to override the image cache)